app/components/Meetup.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+class Meetup extends React.Component {
+  componentDidMount() {
+    const script = document.createElement("script");
+    script.type = "text/javascript";
+    script.src = "https://api.meetup.com/2/events?group_urlname=ReactJS-Denver&callback=parseData";
+    document.body.appendChild(script);
+  }
+
+  render() {
+    return (
+      <div className="meetup-main">
+        <h1>Denver's monthly <br/> React Meetup</h1>
+        <h2>Next Meetup:
+          <span className="meetup-date"> </span>
```

Comment:
When the page loads, there is a "flash" of no date. It would be cool to put a "loading" icon if there is no content.